### DISCUSSION TITLE
github on Android12

### DISCUSSION THREAD

**Discussion by ymtezo** [community/community#62439] [Mobile] [answered]
アンドロイドのローカルフォルダをアンドロイドからGitHubにpull, push, commit する方法はありますか？

**Comment by 5678bgu** [comment]
> アンドロイドのローカルフォルダをアンドロイドからGitHubにpull, push, commit する方法はありますか？

**Comment by h7x4** [comment]
GitHubの公立アプリはその可能性がありませんが、[Termux](https://play.google.com/store/apps/details?id=com.termux&hl=en&gl=US)という端末アプリがあります。Termuxでgitとか[gh](https://cli.github.com/)とかをインストールしたら、pullもpushもcommitもできるはずです。

**Reply by ymtezo** [reply]
教えていただきありがとうございます。
ただし、最新のAndroidのバージョンには非対応みたいです。
他の解決策があればお教えいただきたいです。

**Reply by h7x4** [reply, marked as answer]
[Google Playで発刊するのをやめたらしいです。](https://github.com/termux/termux-app#google-play-store-deprecated)

[ここ](https://github.com/termux/termux-app/releases/tag/v0.118.0)からインストールしたらできるかもしれないけど、

> **NOTICE: Termux may be unstable on Android 12+**

これも書いてあるのでどうなるかよくわかりません。

[このStackoverflow投稿](https://stackoverflow.com/questions/36232183/can-i-commit-files-to-github-from-android-phone)によって`Git Client`を検索すると複数のアプリがあります。私はTermuxしか使ったことないからどっちをおすすめすればいいかわかりません ¯\\\_(ツ)\_/¯

https://play.google.com/store/search?q=git%20client&c=apps

[MGit](https://play.google.com/store/apps/details?id=com.manichord.mgit)は良さそうかな？

**Reply by ymtezo** [reply]
ありがとうございます。本当に助かります！
数日のうちにいくつか試してみます！
